### ISSUE TITLE
how openPDC receives the data of the test signal

### ISSUE THREAD

**Issue by snail526** [GridProtectionAlliance/openPDC#158]

**Comment by snail526** [comment]
hello everyone,

I just contacted openPDC and openHistorian software, downloaded openPDC and openHistorian software, and successfully received test data. My questions are as follows:

I want to know how openPDC receives the data of the test signal. This part of the function corresponds to what content in the source code. (I have downloaded the source code of the GSF framework and openPDC, but I don't know where to start.)

I look forward to receiving your reply and wish you all the best. thank you!
